README.md

```diff
@@ -193,7 +197,7 @@ batcher := gobatcher.NewBatcherWithBuffer(buffer).
 
 - __CapacityInterval__ [DEFAULT: 100ms]: This determines how often the Batcher asks the RateLimiter for capacity. Generally you should leave this alone, but you could increase it to slow down the number of storage Operations required for sharing capacity. Please be aware that this only applies to Batcher asking for capacity, it doesn't mean the rate limiter will allocate capacity any faster, just that it is being asked more often.
 
-- __AuditInterval__ [DEFAULT: 10s]: This determines how often the Target is audited to ensure it is accurate. The Target is manipulated with atomic Operations and abandoned batches are cleaned up after MaxOperationTime so Target should always be accurate. Therefore, we should expect to only see "audit-pass" and "audit-skip" events. This audit interval is a failsafe that if the buffer is empty and the MaxOperationTime (on Batcher only; Watchers are ignored) is exceeded and the Target is greater than zero, it is reset and an "audit-fail" event is raised. Since Batcher is a long-lived process, this audit helps ensure a broken process does not monopolize SharedCapacity when it isn't needed.
+- __AuditInterval__ [DEFAULT: 10s]: This determines how often the Target and Inflight variables are audited to ensure they are accurate. The Target is manipulated with atomic Operations and abandoned batches are cleaned up after MaxOperationTime so Target should always be accurate. Inflight is a channel and abandoned batches are cleaned up after MaxOperationTime so it should also always be accurate. Therefore, we should expect to only see "audit-pass" and "audit-skip" events. This audit interval is a failsafe that if the buffer is empty and the MaxOperationTime (on Batcher only; Watchers are ignored) is exceeded and the Target or Inflight is greater than zero, they are reset and an "audit-fail" event is raised. Since Batcher is a long-lived process, this audit helps ensure a broken process does not monopolize SharedCapacity when it isn't needed or that batches are stalled indefinately.
```

Comment:
Is this used strictly for auditing, or is it also a watchdog? It sounds like a combination of both, since it prunes zombied reservations in addition to raising audit pass/skip/fail events. And since it's manipulating those reservations, but I don't think it takes any steps to pre-empt/terminate those zombies, there should probably be a statement here that indicates that those stalled batches are abandoned but **not** terminated/canceled.

Reply:
I added some verbiage to clarify it abandons watching batches that exceed the maximum time.